addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4492,7 +4492,7 @@ registry.sizing = SnippetOptionWidget.extend({
             };
             const bodyMouseUp = function () {
                 $body.off('mousemove', bodyMouseMove);
-                $body.off('mouseup', bodyMouseUp);
+                $body.add($(window)).off("mouseup", bodyMouseUp);
```

Comment:
Because the body is within an iframe, events inside it bubble only until the root of the iframe, and events inside the window are not captured within the iframe.
Event propagation is confined within the DOM of each individual document.